Changelog.md

```diff
@@ -4,6 +4,7 @@
 
 * motoko (`moc`)
 
+  * Emit new ICP custom section 'motoko:compiler' with compiler release or revision in UTF8 (e.g. "0.6.21"). Default is private (#3091).
```

Comment:
what do you mean by "private"?

Reply:
ICP sections can be public or private. Private means only accessible from controller (but the system can still peek at it I assume). Otherwise people could search for canisters compiled with moc versions with known vulnerabilities.

Reply:
https://github.com/dfinity/interface-spec/blob/master/spec/index.adoc#canister-information

Reply:
Yeah, ok, `icp:private`